packages/typescriptlang-org/src/components/search/ResultRow.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react"
+
+import { cx } from "../../lib/cx"
+import { JoinedPackage } from "./types"
+
+export type ResultRowprops = {
+  installer: [string, string]
+  joinedPackage: JoinedPackage
+}
+
+export const ResultRow: React.FC<ResultRowprops> = ({
```

Comment:
I honestly did try to keep things generally in the one file per the existing convention... but it got very big and I had a hard time keeping track of what's in it, so I gave into my inner craving and split it all out. 😞

Reply:
Totally reasonable 👍🏻